packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -46,7 +46,7 @@
         @click.stop="reset()"
       />
       <va-icon
-        v-if="loading"
+        v-if="loading || isInnerLoading"
```

Comment:
This can be `loadingComputed` from `useValidation` hook and used across all components.

Reply:
@m0ksem got it, I'll make separated validation functions.